<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.378855 rgba(235, 84, 56, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));}</string>
   </property>
   <widget class="QLabel" name="Header">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Do you want to save B/W image?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="No">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>92</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
font: 57 18pt &quot;Noto Sans&quot;;
color:rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Enter the file name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Yes">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>92</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePath">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>221</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Save location</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
